torchtext/transforms.py

```diff
@@ -133,12 +138,33 @@ def __init__(
         self._label_vocab = Vocab(torch.classes.torchtext.Vocab(label_names, 0))
         self._label_names = self._label_vocab.get_itos()
 
-    def forward(self, labels: Union[str, List[str]]) -> Union[int, List[int]]:
-        if torch.jit.isinstance(labels, List[str]):
-            return self._label_vocab.lookup_indices(labels)
+    def forward(self, input: Any) -> Any:
+        if torch.jit.isinstance(input, List[str]):
+            return self._label_vocab.lookup_indices(input)
+        elif torch.jit.isinstance(input, str):
+            return self._label_vocab.__getitem__(input)
         else:
-            return self._label_vocab.__getitem__(labels)
+            raise TypeError("Input type not supported")
 
     @property
     def label_names(self) -> List[str]:
         return self._label_names
+
+
+class Truncate(Module):
```

Comment:
The implementation LGTM. 

However, since this new transform is not used by the other change, I think that the addition of `Truncate` can be in a separate, self-contained PR. Splitting it will give better UX when users reading release note and looking into the commit.

Reply:
Again good suggestion. Actually this whole idea of changing to `Any` became suddenly apparent after I started adding this transform. And then I just kept all the changes :). I will do another PR to add `Truncate` and remove it here.